src/foundry/client/pixi/placeables/sound.d.ts

```diff
@@ -1,9 +1,13 @@
 import type { ConfiguredDocumentClassForName } from "../../../../types/helperTypes";
 import type { DocumentModificationOptions } from "../../../common/abstract/document.mjs";
 
+export {};
```

Comment:
I'm not going to ask you to change anything because it won't matter once we change to modules but just FYI the reason why we have `export {};` sometimes is because ANY import/export makes `declare global` work the way you're used to. In this file `export {};` isn't needed because there's an import right above it.

No action necessary. Don't bother changing anything.